README.md

```diff
@@ -381,6 +382,58 @@ class Error:
     message: str  # Error message
 ````
 
+### Online Refund Payment
+#### RefundRequest - Refund Request
+```python
+@dataclass
+class RefundRequest:
+    store_id: str
+    payment_id: str
+    amount: float
+```
+
+Request parameters:
+- `store_id` - the ID of your assigned store
+- `payment_id` - the unique identifier created by Kody
+- `amount` - amount as a 2.dp decimal number, such as `"1.00"`
+
+
+#### RefundResponse - Refund Response
+
+````python
+class RefundStatus(Enum):
+    PENDING = 0
+    REQUESTED = 1
+    FAILED = 2
+    UNRECOGNIZED = -1
```

Comment:
Are you able to find out what is the final stages?

Reply:
Do you mean the final stages of refund ?

Reply:
Yes, the final status for a refund, which ones are transient and they (developer) could keep refreshing, and the ones that are immutable (like the last two.
I'm not sure if PENDING and REQUESTED are clear enough. Everyone is going to ask which was makes them sure that the funds were refunded.

Reply:
The point is that in the Payment Status, there is the entry SUCCESS = 2, which can be interpreted as a final status. But on a refund the REQUESTED has the same behaviour. We internally know that this is final and successful refund, but for a developer it's not obvious and we would need to document it, or even rename the enum entry as `SUCCESS_REQUESTED`

Reply:
I understand. It might be easier to explain in the documentation since paying and refunding have different "conclusion timings". Refund might take a while. Now that I think of it, I am not sure if we will ever get PENDING during a refund request. It will probably be either a success or fail. Only get payment details would return more granular states.

Reply:
Ok, let's focus on the documentation then